charts/studio/templates/_helpers.tpl

```diff
@@ -88,6 +88,15 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
+{{- define "studio-blobvault-nginx.labels" -}}
```

Comment:
```suggestion
{{- define "studio-blobvault.labels" -}}
```
Let's just call the deployment `studio-blobvault`. It will make the application easier to understand. Users shouldn't worry about what software is serving the blobvault